<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="5" passed="4">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="104367" started-at="2018-10-24T09:52:37Z" finished-at="2018-10-24T09:54:21Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="104367" started-at="2018-10-24T09:52:37Z" finished-at="2018-10-24T09:54:21Z">
      <class name="WSM_LOGOUT.wsmLogout">
        <test-method status="PASS" signature="launchBrowser()[pri:0, instance:WSM_LOGOUT.wsmLogout@bd8db5a]" name="launchBrowser" is-config="true" duration-ms="66433" started-at="2018-10-24T09:52:37Z" finished-at="2018-10-24T09:53:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method status="PASS" signature="TC01_MessageLogout()[pri:0, instance:WSM_LOGOUT.wsmLogout@bd8db5a]" name="TC01_MessageLogout" duration-ms="128" started-at="2018-10-24T09:53:44Z" finished-at="2018-10-24T09:53:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC01_MessageLogout -->
        <test-method status="PASS" signature="TC02_LoginScreen()[pri:1, instance:WSM_LOGOUT.wsmLogout@bd8db5a]" name="TC02_LoginScreen" duration-ms="22" started-at="2018-10-24T09:53:44Z" finished-at="2018-10-24T09:53:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC02_LoginScreen -->
        <test-method status="PASS" signature="TC03_ErrorMessage()[pri:2, instance:WSM_LOGOUT.wsmLogout@bd8db5a]" name="TC03_ErrorMessage" duration-ms="1775" started-at="2018-10-24T09:53:44Z" finished-at="2018-10-24T09:53:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC03_ErrorMessage -->
        <test-method status="PASS" signature="TC04_LogoutAtAnyPage()[pri:3, instance:WSM_LOGOUT.wsmLogout@bd8db5a]" name="TC04_LogoutAtAnyPage" duration-ms="34684" started-at="2018-10-24T09:53:45Z" finished-at="2018-10-24T09:54:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC04_LogoutAtAnyPage -->
        <test-method status="FAIL" signature="TC05_NavigateAfterLogout()[pri:4, instance:WSM_LOGOUT.wsmLogout@bd8db5a]" name="TC05_NavigateAfterLogout" duration-ms="1184" started-at="2018-10-24T09:54:20Z" finished-at="2018-10-24T09:54:21Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [https://wsm.framgia.vn/en] but found [https://wsm.framgia.vn/en/dashboard/user_settings/edit]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [https://wsm.framgia.vn/en] but found [https://wsm.framgia.vn/en/dashboard/user_settings/edit]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at WSM_LOGOUT.wsmLogout.TC05_NavigateAfterLogout(wsmLogout.java:82)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC05_NavigateAfterLogout -->
      </class> <!-- WSM_LOGOUT.wsmLogout -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
